scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/language/tabularDataDefinition/TabularDataDefinitionParser.scala

```diff
@@ -97,7 +98,8 @@ object TabularDataDefinitionParser extends ExpressionParser {
                   errorMessage =
                     s"Column has '${coordinates.columnName.aType.getSimpleName}' type but its value cannot be converted to the type."
```

Comment:
The error message could be better - user just has to know that there is a type mismatch between cell and column, doesn't need to know about conversion.